lessons/de/chapter_5.yaml

```
@@ -0,0 +1,224 @@
+- title: Kapitel 5 - Eigentums- und Kreditdaten
+  content_markdown: >
+    Rust verfügt im Vergleich zu anderen Programmen über ein einzigartiges Paradigma für die
+    Speicherverwaltung Sprachen. Wir werden uns das ansehen
+
+    die Verhaltensweisen und Validierungen des Compilers nacheinander, also nicht
+    überwältigend. Es ist wichtig, sich daran zu erinnern 
```

Remove trailing whitespace:
```suggestion
    überwältigend. Es ist wichtig, sich daran zu erinnern
```